src/view/mainwidget.cpp

```diff
@@ -1,22 +1,50 @@
 #include "mainwidget.h"
+#include "persistence/persistence.h"
+#include <QDebug>
 
 
 MainWidget::MainWidget(QWidget *parent) : QWidget(parent)
 {
-    model_ = new Model();
+    p_ = new Persistence();
+    model_ = new Model(p_);
```

Comment:
I think we should put this in the main.cpp to create dependency injection. It would be better for View testing.

Reply:
Great catch, will patch it out!